docs/DeveloperGuide.md

```diff
@@ -1,25 +1,196 @@
+store object to get an ArrayList of reviews, then calling `getAverageRating()` to get the 
+average rating of the store. After that, `getStoreName()` is also called to get the store
+name of the store. These parameters are then passed to the ui object to be displayed by calling 
+`Ui.showReviews()`
+
+### Reset Store Feature
+#### Implementation
+![DisplayMenus Sequence Diagram](./img/ResetStore.png)
+
+To reset the store index in nusFoodReviews, `ResetStoreCommand#execute()` is called, 
+passing in an ArrayList of canteens, and the ui object instantiated in NusFoodReviews.
+
+When `ResetStoreCommand` is first called, we pass the main NusFoodReviews object to the 
+constructor. This allows the `Command` to interact with the main object when `execute` is called.
+
+### Home Feature
+#### Implementation
+![DisplayMenus Sequence Diagram](./img/HomeCommand.png)
+
+To reset the store and canteen index in nusFoodReviews, `HomeCommand#execute()` is called,
+passing in an ArrayList of canteens, and the ui object instantiated in NusFoodReviews.
+
+When `HomeCommand` is first called, we pass the main NusFoodReviews object to the
+constructor. This allows the `Command` to interact with the main object when `execute` is called.
+
+
+### Admin Capabilities
+#### Implementation
+![Admin Sequence Diagram](./img/Admin.png)
+
+Once admin is verified in NusFoodReviews, `Parser#parseAdminCommand()` is called.
+A switch class is then used to determine the command to instantiate.
+To add a new canteen, the user must enter '2'.
+
+<!--can someone add switch case and separate refs for each switch case maybe-->
+
+### [Admin] Add Canteen
+#### Implementation
+![AddCanteen Sequence Diagram](./img/AddCanteen.png)
+
+<!--this part should be in the switchcase ref-->
+`Ui#showAddCanteen()` is called to display the add canteen prompt.
+User input for canteen name will then be read using `Ui#readCommand()`.
+A new AddCanteenCommand object is instantiated, with the canteen name passed into the constructor.
+`Parser#parseAdminCommand()` returns the AddCanteenCommand object.
+<!--this part should be in the switchcase ref-->
+
+To add a canteen, `AddCanteenCommand#execute()` is called, passing in 
+an ArrayList of canteens and the Ui object instantiated in NusFoodReviews.
+
+A new Canteen object is then instantiated, and added to the ArrayList of canteens.
+`Ui#showAddCanteenSuccess()` is called to display canteen added confirmation.
+
+### [Admin] Add Store
+#### Implementation
+![AddCanteen Sequence Diagram](./img/AddStore.png)
+
+
+To add a store, `AddStoreCommand#execute()` is called, passing in
+an ArrayList of canteens and the Ui object instantiated in NusFoodReviews.
+
+A new Store object is then instantiated, and added to the Canteen's ArrayList of stores.
+`Ui#printStoreAdded()` is called to display store added confirmation.
+
+
+### [Admin] Delete Canteen
+#### Implementation
+![DeleteCanteen Sequence Diagram](./img/DeleteCanteen.png)
```

Comment:
I have a question for this function here: Does deleting the canteen delete the review of the canteen at the same time?